app/components/post_list/post_list.tsx

```diff
@@ -17,6 +17,8 @@ import {useServerUrl} from '@context/server';
 import {useTheme} from '@context/theme';
 import {getDateForDateLine, isCombinedUserActivityPost, isDateLine, isStartOfNewMessages, isThreadOverview, preparePostList, START_OF_NEW_MESSAGES} from '@utils/post_list';
 
+import CompassIcon from '../compass_icon';
```

Comment:
Let's make this follow the same pattern as the other imports

```suggestion
import CompassIcon from '@components/compass_icon';
```
 And then sort it accordingly